server/index.js

```diff
@@ -22,6 +22,10 @@ const db = require("./lib/in-memory-db");
 // require it and pass the `db` parameter immediately:
 const DataHelpers = require("./lib/data-helpers.js")(db);
 
+// This import is an immediately invoked function expression
+// which will update the dates for the initial tweets (data-files/initial-tweets.json).
```

Comment:
You can remove line 25 and remove `which will` from line 26.

Side note: sorry, still not an IIFE. It's missing the "immediately" portion. :)

Reply:
Done and thank you - I'm learning a lot from this so it's appreciated!